deploy/kube-config/influxdb/grafana-image/Dockerfile

```diff
@@ -0,0 +1,15 @@
+#
```

Comment:
Can we move all the grafana image related files to a top-level `grafana` directory similar to how InfluxDB is managed?